lib/widget/circle_button.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:flutter/material.dart';
+import 'package:survey_flutter_ic/gen/assets.gen.dart';
+
+class CircleButton extends StatelessWidget {
```

Comment:
The widget name is too generic, can we update it to `WhiteRightArrowButton`?

Reply:
@manh-t Nice. I like this name 👍